### PR TITLE
Chart js (Workshop Bounty)

### PR DESCRIPTION
**Pull request by wollygfx**

### CODE REVIEW

File: workshops/Chart Js/README.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
+---
+name: Chart JS
+description: Create any kind of chart for your website
+author: '@wollygfx'
+img: ''
```
Comment (sampoder) [inline workshops/Chart Js/README.md:5]:
```suggestion
img: 'https://cloud-g96wt9icu.vercel.app/0screen_recording_2020-11-19_at_4.38.37_pm.gif'
```

File: workshops/Chart Js/README.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
+---
+name: Chart JS
+description: Create any kind of chart for your website
+author: '@wollygfx'
+img: 'https://cloud-g96wt9icu.vercel.app/0screen_recording_2020-11-19_at_4.38.37_pm.gif'
+
+---
+
+
```
Comment (sampoder) [inline workshops/Chart Js/README.md:9]:
```suggestion
```

File: workshops/Chart Js/README.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
+---
+name: Chart JS
+description: Create any kind of chart for your website
+author: '@wollygfx'
+img: 'https://cloud-g96wt9icu.vercel.app/0screen_recording_2020-11-19_at_4.38.37_pm.gif'
+
```
Comment (sampoder) [inline workshops/Chart Js/README.md:6]:
```suggestion

```

**Review by sampoder — CHANGES_REQUESTED** [review]
Hey!

Thanks for submission, this is a fairly well written workshop and has great potential. However, I think it still has a way to go. 

My main concern with this workshop is that the end product isn't satisfactory enough. Would it be possible for you to build on top of the stuff you learn in this workshop to build a more complex product?

Hope that makes sense,

Sam 🚀

**Comment by wollygfx** [comment]
Hey @sampoder! – Sorry, I feel kinda bad because I've been mentioning you a lot of times today, hope you don't mind 💯 

So, regarding to what you said: 

> My main concern with this workshop is that the end product isn't satisfactory enough. Would it be possible for you to build on top of the stuff you learn in this workshop to build a more complex product?

I totally agree and I have an idea to fix this:

> 💡 The idea: A chart generator in which, you just put your data in like some sort of inputs, then you choose what type of chart you want to generate, and finally it generates a chart that you can use.

I would love if you could give me your thoughts on this. 😄

**Comment by sampoder** [comment]
sounds like a great plan!

**Pull request closed without merging**